parsers/verilog.c

```diff
@@ -1577,8 +1580,20 @@ static int tagNameList (tokenInfo* token, int c)
 	{
 		c = processType(token, c, &kind);	// update token and kind
 
-		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`')
+		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`' || c == '(')
 		{
+			if(c == '('){
```

Comment:
we don't use this style, putting `{` at the end of line IN THIS FILE.

```
if (c == '(')
{
   ...
```
is expected.